cfgov/v1/jinja2/v1/includes/organisms/footer.html

```diff
@@ -182,11 +182,11 @@
                 <ul class="o-footer_list m-list">
                     <li class="m-list_item">
                         <a class="m-list_link a-link"
-                           href="{{ _('https://usa.gov/') }}">{{ _('USA.gov') }}</a>
+                           href="{{ _('https://www.usa.gov/') }}">{{ _('USA.gov') }}</a>
```

Comment:
Agh, unfortunately I missed that this URL was being passed through the Django translation framework, which means that we would need to edit those files as well if we change this here.

I am happy to assist with that, but there is another complication: [the Spanish translations for this](https://github.com/cfpb/consumerfinance.gov/blob/2aba46f9116a317b2acfa263281fa52d72ffa425/cfgov/v1/locale/es/LC_MESSAGES/django.po#L194-L198) also seem out of date. Currently the link goes to https://gobiernousa.gov/, which redirects to https://www.usa.gov/es/, and the link text says "GobiernoUSA.gov", instead of the default "USA.gov". So this would need an alternate link text, maybe USA.gov/es/?

Reply:
yes on both counts, let's update the link text to "USA.gov/es" and the URL to https://www.usa.gov/es/

Reply:
Just pushed a commit that does the above:

![image](https://github.com/cfpb/consumerfinance.gov/assets/654645/c68e3923-c2de-4ff0-bf43-676928a4097f)